Gemfile

```diff
@@ -10,6 +10,7 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 #
 
 gem "manageiq-gems-pending", ">0", :require => 'manageiq-gems-pending', :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+gem "manageiq-loggers", ">0", :require => false, :git => "https://github.com/agrare/manageiq-loggers", :branch => "add_journald_logger"
```

Comment:
TODO Switch to master once https://github.com/ManageIQ/manageiq-loggers/pull/5 is merged

Reply:
Merged

Reply:
:+1: thanks